eng/pipelines/steps/release-build-steps.yml

```diff
@@ -130,6 +130,28 @@ steps:
         start: true
         reason: queued build
 
+    - script: |
+        releasego build-pipeline \
+          -id '1422' \
```

Comment:
A note for others: this is intentionally hard-coded rather than taken from the variable group (like `-id '$(GoReleaseInnerloopPipelineID)'` above) because the indirection isn't serving any purpose and is more obscure.